bluepyopt/tests/test_ephys/test_models.py

```diff
@@ -180,6 +180,23 @@ def test_CellModel_destroy():
     assert 0 == len(sim.neuron.h.CellModel_destroy)
 
 
+@pytest.mark.unit
+def test_lfpycellmodel():
```

Comment:
we're gonna need way more unit tests to test all the new functionality above. @AurelienJaquier , maybe something for you?

Reply:
Sure, I'm on it.